qiita_pet/templates/analysis_results.html

```diff
@@ -11,44 +11,60 @@
 
 
 <div class="row">
-  <h1>Analysis {{aname}}</h1>
-
-  {% if dropped %}
-    <h3>Dropped Samples</h3>
-    These samples were dropped during processing due to rarefaction:
-    <table width="100%">
-    {% for study, samples in viewitems(dropped) %}
-      <tr>
-        <td width="5%">For {{study}}:</td>
-        <td width="80%"><small>{{', '.join(samples)}}</small></td>
-      </tr>
-    {% end %}
-    </table>
-    <br/>
-  {% end %}
+  <h1>&nbsp;Analysis: <u>{{aname}}</u></h1>
 </div>
 
 
 <div class="row" width='100%'>
+  <div class="col-md-12">
+    <div class="panel-group" id="accordion">
+      {% if dropped %}
+        <div class="panel panel-default">
+          <div class="panel-heading">
+            <h3 class="panel-title"><a data-toggle="collapse" data-parent="#accordion" href="#dropped-samples">Dropped Samples</a></h3>
+          </div>
+          <div id="dropped-samples" class="panel-collapse collapse">
+              <div class="panel-body">
+                  {% for data_type, studies in viewitems(dropped) %}
+                    <b><u>{{data_type}}</u></b>
+                    <br/><br/>
+                    {% for s in studies %}
+                      <b>{{s['study']}}:</b>
+                      <br/>
+                      Total dropped: {{len(s['samples'])}}
```

Comment:
Can you compute the length of the list and do the `', '.join` in the handler? In previous iterations of the code we avoid to add as much python code in templates as possible.